tests/playwright-test/playwright.ct-reuse.spec.ts

```diff
@@ -168,3 +168,83 @@ test('should work with manually closed pages', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(0);
   expect(result.passed).toBe(3);
 });
+
+test('should clean storage', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright/index.html': `<script type="module" src="/playwright/index.ts"></script>`,
+    'playwright/index.ts': `
+      //@no-header
+    `,
+
+    'src/reuse.test.tsx': `
+      //@no-header
+      import { test, expect } from '@playwright/experimental-ct-react';
+      let lastContextGuid;
+
+      test('one', async ({ context, page }) => {
+        lastContextGuid = context._guid;
+
+        await page.evaluate(async () => {
+          localStorage.foo = 'bar';
+          sessionStorage.foo = 'bar';
+        });
+
+        const local = await page.evaluate('localStorage.foo');
+        const session = await page.evaluate('sessionStorage.foo');
+        expect(local).toBe('bar');
+        expect(session).toBe('bar');
+      });
+
+      test('two', async ({ context, page }) => {
+        expect(context._guid).toBe(lastContextGuid);
+        const local = await page.evaluate('localStorage.foo');
+        const session = await page.evaluate('sessionStorage.foo');
+
+        expect(local).toBeFalsy();
+        expect(session).toBeFalsy();
+      });
+    `,
+  }, { workers: 1 });
+});
```

Comment:
You have to `expect(result.exitCode).toBe(0)`, otherwise this doesn't test anything :)

Reply:
yup. done